lib/tests/2wp.js

```diff
@@ -777,6 +777,47 @@ const execute = (description, getRskHost) => {
 
     });
 
+    it('should reject and refund a pegout when fee per kb is above value', async () => {
+
+      // Arrange
+
+      // Create a pegin for the sender to ensure there is enough funds to pegout and because this is the natural process
+      const senderRecipientInfo = await createSenderRecipientInfo(rskTxHelper, btcTxHelper);
+      const peginValueInSatoshis = btcToSatoshis(0.5);
```

Comment:
Maybe we can have a default constant for 0.5 as well?

Reply:
I think there's no need for this. This is not the minimum pegin value or any significan value. This is just to have enough funds on the rsk address. This value could change from test to test. For example, in the future pegout test where we will need to create a lot of pegouts to assert that the Bridge split them, we will probably need to send way more funds. So, that value could change.

Reply:
As long as this value does not repeat in other places, it is ok to me.

Reply:
Left a similar comment in another PR. 

Not a big deal, open to discuss it. If the value won't always be the same then maybe it doesn't make sense to have a constant. But it could maybe at least be a constant within the test to not make it a magic number.

Option 2 is to calculate this value depending on what the test needs. If we are doing a peg-out for `pegoutValue=X`, then this value could be `pegoutValue * 2`

Open to hear other ideas

Reply:
Updated.